draft-ietf-quic-transport.md

```diff
@@ -2526,6 +2571,9 @@ QUIC_CLOSED_CRITICAL_STREAM (0x80000007):
 QUIC_MISSING_PAYLOAD (0x80000030):
 : The packet contained no payload.
 
+QUIC_RECEIVED_RST (0x80000035):
+: Terminating stream because peer sent a RST_STREAM or DISINTEREST.
```

Comment:
"CANCEL" in h2 means "I didn't do anything with this stream", which mightbe nice to keep.